src/jit/importer.cpp

```diff
@@ -7492,17 +7492,6 @@ GenTreePtr                Compiler::impFixupCallLongReturn(GenTreePtr     call,
     unsigned retRegCount = retTypeDesc->GetReturnRegCount();
     // must be a long returned in two registers
     assert(retRegCount == 2);
-
```

Comment:
Function header comment needs to be updated, since this routine is just initializing return type descriptor of a call node.  Also should we rename this method to be representative of its new functionality (e.g. impInitCallReturnTypeDesc())

Note that right now impFixupCallStructReturn() is also initializing return type desc of a call node for unix amd64.  @briansull would need to add similar code for Arm64 case.  In future when we add support for arrm32 this routine would come handy.